File: permission/src/main/java/com/cn/permission/aop/util/EmptyUtil.java
package com.cn.permission.aop.util;

/**
 * @Author: 崔宁 github: https://github.com/Cuinings
 * @Email: 1015597172@qq.com
 * @Date: 2020/6/4 10:02 AM
 * @Description:
 */
public class EmptyUtil {

    public static boolean isEmpty(String str) {
        if (null == str || str.length() == 0)
            return true;
        return false;
    }

    public static boolean isEmpty(String... strings) {
        if (null == strings || strings.length == 0)
            return true;
        return false;
    }
}

File: permission_kotlin/src/main/java/com/cn/permission_kotlin/aop/util/EmptyUtil.kt
package com.cn.permission_kotlin.aop.util

/**
 * @Author:          崔宁 github: https://github.com/Cuinings
 * @Email:          1015597172@qq.com
 * @Date:           2020/6/4 3:18 PM
 * @Description:
 */
class EmptyUtil {

    companion object {
        fun isEmpty(str: String?): Boolean {
            if (str?.isEmpty()!!)
                return true
            return false;
        }

        fun isEmpty(strs: Array<String>?): Boolean {
            if (strs.isNullOrEmpty())
                return true;
            return false
        }
    }
}